index.d.ts

```diff
@@ -69,6 +69,7 @@ declare namespace Eris {
   type MessageContent = string | AdvancedMessageContent;
   type MFALevel = 0 | 1;
   type PossiblyUncachedMessage = Message | { channel: TextableChannel | { id: string; guild?: Uncached }; guildID?: string; id: string };
+  type PossiblyEphemeralMessage = Message | { id: string; flags: number };
```

Comment:
you could explicitly type this as 64, since it will only ever by 64 when ephemeral